src/client/ActionCombineModifierCell.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2025 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { VNode } from "preact";
```

Comment:
Since the combine modifier is not implemented yet, shall this action script be removed from this PR. Might be better to have it in a separate branch for implementing the combine modifier.